op-validator/pkg/validations/addresses.go

```diff
@@ -13,8 +13,9 @@ const (
 
 var addresses = map[uint64]map[string]common.Address{
 	11155111: {
-		VersionV180: common.HexToAddress("0x2A788Bb1D32AD0dcEC1A51B7156015Aa90548d8C"),
-		VersionV200: common.HexToAddress("0x34FFEEF9D42E0EF0d999fBF01E006f745083Fd9b"),
+		// Both versions below bootstrapped on 02/23/2025 using OP Deployer.
+		VersionV180: common.HexToAddress("0x0a5bf8ebb4b177b2dcc6eba933db726a2e2e2b4d"),
```

Comment:
Can we have a mainnet version of v180 deployed also, or are we holding off on that for any reason?

Reply:
we keep changing the contracts, so before we spend money on the mainnet deployment i'd like the contracts to be implementation-locked.